frontend/public/components/secrets/create-secret/types.ts

```diff
@@ -0,0 +1,13 @@
+// TODO Follow on: Move Type definitions for CreateSecret here
+
```

Comment:
Ticket: CONSOLE-4291

Reply:
I think moving shared types into a types file is a good idea. Since we have a story to cover those changes, should we consider them out of scope for this story?